SharedPackages/DesignResourcesKit/Sources/DesignResourcesKit/Colors/SharedPalettes/DesertColorPalette.swift

```diff
@@ -22,8 +22,6 @@ import SwiftUI
 
 /// Autogenerated from [Figma](https://www.figma.com/design/lowzSNK3XysASQ1urGCZFp/Browser-Color-Themes?node-id=504-17677&m=dev)
 struct DesertColorPalette: SharedColorPaletteDefinition {
```

Comment:
⬇️  **Autogenerated**, [sourced from the latest Figma Json Import](https://app.asana.com/1/137249556945/project/1201048563534612/task/1211371913634199?focus=true)